salsah/src/public/js/jquery.resadd.js

```diff
@@ -517,6 +517,7 @@
 												}
 											}
 											tmpele.attr('placeholder', placeholderText);
+											tmpele.attr('autocomplete', 'off');
```

Comment:
Ok, now I understand. I think it's good to deactivate the standard autocomplete because this could confuse the user.